src/ocamlorg_frontend/pages/outreachy.eml

```diff
@@ -0,0 +1,139 @@
+
+let render (metadata : Ood.Outreachy.t list)
+= 
+Layout.render
+~title:"OCaml Outreachy Internships"
+~description:"Records Of OCaml Community Past Outreachy Internship Projects"
+~canonical:Url.outreachy @@
+<div class="intro-section-simple">
+    <div class="container-fluid">
+        <div class="text-center w-full lg:w-2/3 m-auto">
+            <h1 class="font-bold mb-6">OCaml Outreachy Internships</h1>
+            <p>This is record of all past OCaml Community Outreachy Internship Projects.</p>
+        </div>
+    </div>
+</div>
+
+<div class="container relative mx-auto px-4 items-center" x-on:keydown.escape="isModalOpen=false" x-data="{'isModalOpen': false, 'slug': null}">
+        <% metadata |> List.iter (fun (rounds : Ood.Outreachy.t) -> %>
+            <div id="name" class="block  border-b border-[#EF8316] p-6 text-left my-2">
+                <h2 class="font-bold"><%s rounds.name %></h2>
+            </div>
+            <div class="grid grid-cols-1 auto-rows-max md:grid-cols-2 xl:grid-cols-3 gap-10">
+                <% rounds.projects |> List.iter (fun (project : Ood.Outreachy.project) -> %>
```

Comment:
Hey, a small suggestion here: maybe you can use `List.iter_i` here and remove the `number` field? I suspect keeping this field up-to-date might become a significant maintenance burden.

(feel free to ignore my suggestion :p)